app/common/constants/urls.ts

```diff
@@ -0,0 +1,3 @@
+export const forms = "/forms"
+export const login = "/forms/login"
+export const newWallet = "/forms/wallet"
```

Comment:
IMO, this file should not be placed in common folder but in renderer instead given that the main process won't be using these urls